### PR TITLE
Fixes to strings + Hospitality

### PR DESCRIPTION
**Pull request by ZnogyroP**
## Description
Fixes a few battle strings, specifically ones that included "the" before an attacker or defender with a prefix, producing effects like "The Wild Sunkern's Solar Power took its toll!" Also fixed Hospitality's message, which for some reason was hardcoded to refer to Sinistcha, even though Poltchageist can also have it.

## **Discord contact info**
znogyrop

### CODE REVIEW

File: src/battle_util.c [hunk 1]
```diff
@@ -4653,6 +4653,7 @@ u32 AbilityBattleEffects(u32 caseID, u32 battler, u32 ability, u32 special, u32
             if (!gSpecialStatuses[battler].switchInAbilityDone && IsDoubleBattle() && gBattleMons[partner].hp < gBattleMons[partner].maxHP)
             {
                 gBattlerTarget = partner;
+				gBattlerAttacker = battler;
```
Comment (Bassoonian) [inline src/battle_util.c:4656]: spaces instead of tabs

Comment (AlexOn1ine) [inline src/battle_util.c:4656]: tabs

**Review by Bassoonian — COMMENTED** [review]
Make sure to update the expected messages in test files accordingly, too, or it won't pass the check :)

**Review by AlexOn1ine — COMMENTED** [review]
Since you've changed the string some tests fail now.

**Review by AlexOn1ine — APPROVED** [review]

**Pull request merged**